### PR TITLE
Show link to latest version in examples

### PR DESCRIPTION
**Pull request by ahocevar**
Together with openlayers/ol3#5332, this fixes openlayers/ol3#5235.

### CODE REVIEW

File: en/v3.0.0/resources/example-behaviour.js [hunk 1]
```diff
@@ -1,4 +1,47 @@
 ;(function() {
+
+  // show outdated version warning
+  var container = document.getElementsByClassName('container-fluid')[0];
```
Comment (tschaub) [inline en/v3.0.0/resources/example-behaviour.js]: Our main template has an element with `id="title"`.  Would it make sense to insert the warning before that instead?  Not sure that is more or less reliable.  So no need to change if you prefer not to.

**Comment by marcjansen** [comment]
Thanks for the continued work on this, @ahocevar.

Looks good to me, please merge.

File: en/v3.0.0/resources/example-behaviour.js (hunk 1 above)
Comment (tschaub) [inline en/v3.0.0/resources/example-behaviour.js]: I see now you are porting over the changes from https://github.com/openlayers/ol3/pull/5332 to old examples.  So you can ignore the comment above.

**Comment by tschaub** [comment]
Agreed.  Thanks for pushing this through to the finish.

**Comment by ahocevar** [comment]
Thanks for the reviews!

**Pull request merged**